backend/satellite_tools/mgr-sign-metadata-ctl

```diff
@@ -0,0 +1,296 @@
+#!/bin/bash
+
+set -e
+
+function usage {
+    echo "Usage: mgr-sign-metadata-ctl [enable <keyid> | disable | check-config | check-channels]"
+    echo "  enable KEYID     Enable GPG signing of repository metadata using the provided KEYID."
+    echo "                   Requires a valid key with the given KEYID to exist in the default keyring."
+    echo "  disable          Disable GPG signing of repository metadata."
+    echo "  check-channels   Check if the cached channels repodata is signed."
+    echo "  check-config     Check configuration and report any problems. This is the default option."
+    echo "  regen-metadata   Restart services and schedule metadata regeneration."
+}
+
+function check_key_in_keyring {
+    if gpg --list-keys | grep --quiet -F "$1"; then
+        echo "OK. Found key $1 in keyring."
+    else
+        echo "ERROR. Key $1 not found in keyring. Generate or import a key pair and then rerun this command."
+        exit 2
+    fi
+}
+
+function regenerate_metadata {
+    echo -n "Restarting Tomcat..."
+    systemctl restart tomcat
+    echo "done."
+    echo -n "Restarting Taskomatic..."
+    systemctl restart taskomatic
+    echo "done."
+    echo
+    echo "Scheduling repo metadata regeneration..."
+    read_spacecmd_user_pass
+    spacecmd -q -u $SPACECMD_USER -p $SPACECMD_PASS "softwarechannel_regenerateyumcache -f *"
```

Comment:
There is no option "-f" supported by regenerateyumcache. Also the XMLRPC function only allowes a restriction of channels.

Reply:
Ok, sorry. You have implemented it here in this PR.